plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -828,6 +847,43 @@ export async function createRouter(
       });
 
       res.status(200).json({ results });
+    })
+    .get('/v2/template-filters/built-in', async (_req, res) => {
```

Comment:
Not sure that there's value of splitting these up by `built-in` or not. I think that it might be best to just have a `/v2/templating-extensions` or something that can list all of them rather than splitting up by page.

Reply:
My thought process was that a user could theoretically override a built-in filter by specifying a custom filter with the same name, which might cause confusion. I can collapse, though.